.buildkite/dagster-buildkite/dagster_buildkite/package_spec.py

```diff
@@ -190,26 +192,22 @@ def build_steps(self) -> List[GroupStep]:
                         )
                     )
 
-        if self.run_mypy:
-            steps.append(
-                build_tox_step(
-                    self.directory,
-                    "mypy",
-                    base_label=base_name,
-                    command_type="mypy",
-                    python_version=supported_python_versions[-1],
-                    skip_reason=self.skip_reason,
-                )
-            )
-
         emoji = _PACKAGE_TYPE_TO_EMOJI_MAP[self.package_type]  # type: ignore[index]
-        return [
-            GroupStep(
-                group=f"{emoji} {base_name}",
-                key=base_name,
-                steps=steps,
-            )
-        ]
+        if len(steps) >= 2:
+            return [
+                GroupStep(
+                    group=f"{emoji} {base_name}",
+                    key=base_name,
+                    steps=steps,
+                )
+            ]
+        elif len(steps) == 1:
```

Comment:
What are we trying to guard against here?

Reply:
We return a `CommandStep` instead of a `GroupStep` if there's only one step, to avoid having to unnecessarily click through a group layer in BK.